insights/parsers/cmdline.py

```diff
@@ -59,8 +59,15 @@ def parse_content(self, content):
         for el in line.strip().split(None):
             # Thanks to strip and split, el always contains something.
             if "=" not in el:
-                cmdline_properties[el] = None
+                if cmdline_properties.get(el) is None:
+                    cmdline_properties[el] = None
+                if isinstance(cmdline_properties[el], list):
+                    cmdline_properties[el].append(None)
             else:
                 (key, value) = el.split("=", 1)
```

Comment:
Maybe we should always make things lists so that the interface is consistent.  They way this works now you have to check if the value is a list or a scalar value in the calling code.

Reply:
I realize that this comment was applied to the wrong side of the diff.